packages/admin/src/pages/super-admin-dashboard/manage-departments/delete/[id]/index.page.tsx

```diff
@@ -55,12 +59,9 @@ const DeleteDepartmentPage = ({
             >
               Delete department
             </button>
-            <a
-              className="govuk-link"
-              href={`/super-admin-dashboard/manage-departments/edit/${id}`}
-            >
-              Cancel
-            </a>
+            <Link href={`/super-admin-dashboard/manage-departments/edit/${id}`}>
```

Comment:
Use CustomLink instead. We've had many an issue for using nexts link in the past, it trys to do fancy stuff but ends up breaking things in the process like initing the govuk javascript package on pageload